Weird Frontiers: Update sheet.json

bigmc10 opened the pull request:
## Changes / Comments

Enable legacy to avoid CSE




## Roll20 Requests

Comments are very helpful for reviewing the code changes. Please answer the relevant questions below in your comment.

- [X ] Does the pull request title have the sheet name(s)? Include each sheet name.
- [ ] Is this a bug fix?
- [ ] Does this add functional enhancements (new features or extending existing features) ?
- [ ] Does this add or change functional aesthetics (such as layout or color scheme) ? 
- [ ] If changing or removing attributes, what steps have you taken, if any, to preserve player data ?
- [ ] If this is a new sheet, did you follow [Building Character Sheets standards](https://wiki.roll20.net/Building_Character_Sheets#Roll20_Character_Sheets_Repository) ?

If you do not know English. Please leave a comment in your native language.

roll20deploy commented:
[Character Sheet Info](https://app.roll20.net/managesheets/WeirdFrontiers/roll20-character-sheets) *Roll20 Internal Use only.*

mperes commented:
Dear @bigmc10, your pull request contains changes to multiples sheets. Please split your PR into one request per sheet. Thanks fo all the work you do!

bigmc10 commented:
I forgot I had created prototype files in my branch for a new sheet that doesn't exist yet. I deleted those files now in my branch and updated from master. Can the pull of the one file change under the WeirdFrontiers folder proceed? Do I need to submit a new pull request? Will the addition and subsequent deletion of the other files still show up as changes and cause the same problem?
thanks,Scott

    On Monday, May 17, 2021, 02:49:02 PM EDT, Miguel Peres ***@***.***> wrote:  
 
 


Dear @bigmc10, your pull request contains changes to multiples sheets. Please split your PR into one request per sheet. Thanks fo all the work you do!

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or unsubscribe.
